WORKING_GROUPS.md

```diff
@@ -0,0 +1,249 @@
+# Node.js Community Working Groups
+
+Node.js Community Working Groups are autonomous projects created by the
+[Community Committee (CommComm)][./GOVERNANCE.md].
+
+Working Groups can be formed at any time but must be ratified by the CommComm.
+Once formed the work defined in the Working Group charter is the
+responsibility of the WG rather than the CommComm.
+
+It is important that Working Groups are not formed pre-maturely. Working
+Groups are not formed to *begin* a set of tasks but instead are formed
+once that work is already underway and the contributors
+think it would benefit from being done as an autonomous project.
+
+If the work defined in a Working Group's charter is complete, the charter
+should be revoked.
+
+A Working Group's charter can be revoked either by consent of the Working
+Group's members or by a CommComm vote. Once revoked, any future work that arises
+becomes the responsibility of the CommComm.
+
+## Joining a WG
+
+To find out how to join a working group, consult the (GOVERNANCE.md)[./GOVERNANCE.md] in
+the working group's repository.
+
+## Starting A Community Working Group
+
+A Working Group is established by first defining a charter that can be
+ratified by the CommComm. A charter is a *statement of purpose*, a
+*list of responsibilities* and a *list of initial membership*.
+
+A working group needs 3 initial members. These should be individuals
+already undertaking the work described in the charter.
+
+The list of responsibilities should be specific. Once established, these
+responsibilities are no longer governed by the CommComm and therefore should
+not be broad or subjective. The only recourse the CommComm has over the working
+group is to revoke the entire charter and take on the work previously
+done by the working group themselves.
+
+If the responsibilities described in the charter are currently
+undertaken by another WG then the charter will additionally have to be
+ratified by that WG.
+
+You can submit the WG charter for ratification by sending
+a Pull Request to this document, which adds it to the
+list of current Working Groups. Once ratified the list of
+members should be maintained in the Working Group's
+README.
+
+## Bootstrap Governance
+
+Once the CommComm ratifies a charter the WG inherits the following
+documentation for governance, contribution, conduct and an MIT
+LICENSE. The WG is free to change these documents through their own
+governance process, hence the term "bootstrap."
+
+```markdown
+### *[insert WG name]* Working Group
+
+The Node.js *[insert WG name]* is jointly governed by a Working Group (WG)
+that is responsible for high-level guidance of the project.
+
+The WG has final authority over this project including:
```

Comment:
CommComm projects usually won't have technical direction, which is the first item int his list. This list isn't really a fit for the structure of CommComm projects - and is probably a good reason that we've not codified this yet.